etcd3/client_facade.py

```diff
@@ -0,0 +1,42 @@
+from etcd3 import client, exceptions
```

Comment:
Represents the abstraction of retry logic i.e. it will catch etcd exceptions and retry the operation until every endpoints has been failed.

I really dont like this. Do you guys have an idea on how to put this into the client itself?